### PR TITLE
Connect webpage to tabmodel permanently to track url changes

### PR DESCRIPTION
**Pull request by rojkov**

### CODE REVIEW

File: src/declarativewebcontainer.cpp [hunk 1]
```diff
@@ -141,7 +141,7 @@ void DeclarativeWebContainer::setWebPage(DeclarativeWebPage *webPage)
             connect(m_webPage, SIGNAL(canGoBackChanged()), this, SIGNAL(canGoBackChanged()), Qt::UniqueConnection);
             connect(m_webPage, SIGNAL(canGoForwardChanged()), this, SIGNAL(canGoForwardChanged()), Qt::UniqueConnection);
             connect(m_webPage, SIGNAL(urlChanged()), this, SIGNAL(urlChanged()), Qt::UniqueConnection);
-            connect(m_webPage, SIGNAL(urlChanged()), this, SLOT(onPageUrlChanged()), Qt::UniqueConnection);
+            connect(m_webPage, SIGNAL(urlChanged()), m_model, SLOT(onUrlChanged()), Qt::UniqueConnection);
```
Comment (rainemak) [inline src/declarativewebcontainer.cpp]: What makes this to permanent? All connections are disconnected from the previous web page a few lines above.

Comment (rojkov) [inline src/declarativewebcontainer.cpp]: They are not. I've added a comment stressing it. The line above disconnect signals from m_webPage to `this` DeclarativeWebContainer instance, not `m_model`.

Comment (rainemak) [inline src/declarativewebcontainer.cpp]: Aah... true... it was disconnect(this).. not disconnect(this, 0, 0, 0);

My bad.

File: tests/auto/tst_dbmanager/tst_dbmanager.pro [hunk 2]
```diff
@@ -1,5 +1,6 @@
 TARGET = tst_dbmanager
```
Comment (rainemak) [inline tests/auto/tst_dbmanager/tst_dbmanager.pro]: Is this accidentally here or really needed for tst_dbmanager.pro?

Needed at least for tst_declarativetabmodel.pro and tst_webview.pro I'd think. Clean build is failing.

Comment (rainemak) [inline tests/auto/tst_dbmanager/tst_dbmanager.pro]: tst_declarativebookmarkmodel.pro could have
```
+NO_COMMON_INCLUDES=1
 include(../test_common.pri)
```
tst_declarativehistorymodel.pro requires also declarativewebpage_mock.pri include

Comment (rojkov) [inline tests/auto/tst_dbmanager/tst_dbmanager.pro]: Ah, the assumption that it's enough to remove Makefiles was wrong... Fixed.

For some reason tst_dbmanager fails to build without the mock, but tst_webview on the opposite doesn't need the mock as it uses the real class.

**Comment by rainemak** [comment]
LGTM! Thanks @rojkov

**Pull request merged**